pom.xml

```diff
@@ -12,6 +12,36 @@
   <name>my-web-app Maven Webapp</name>
   <url>http://maven.apache.org</url>
   <dependencies>
+    <!-- https://mvnrepository.com/artifact/junit/junit -->
```

Comment:
Do we need this comment?
The same about other comments in pom.xml?

Reply:
- [x] comments deleted